source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,225 @@ Hardware Checklist
+- Number of sites to be deployed
+
+Hardware Requirements and Recommendations
+-----------------------------------------
+
+.. note:: 
+
+   See our `Reference Hardware <https://min.io/product/reference-hardware?ref-docs>`__ page for a curated selection of servers and storage components from our hardware partners.
+
+   MinIO does not provide hosted services or hardware sales.
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - | Sufficient CPU cores to acheive performance goals for hashing (for example, for healing) and encryption
+       | MinIO recommends Dual Intel® Xeon® Scalable Gold CPUs (minimum 8 cores per socket) or any CPU with AVX512 instructions
+
+   * - :octicon:`circle`
+     - | Sufficient RAM to achieve performance goals based on the number of drives and anticipated concurrent requests (see the :ref:`formula and reference table <minio-requests-per-node>`)
+       | Refer to the information on :ref:`memory allocation <minio-k8s-production-considerations-memory>` for recommended RAM amounts 
+
+   * - :octicon:`circle`
+     - Four nodes or servers
+
+   * - :octicon:`circle`
+     - | SATA/SAS drives for capacity and NVMe SSDs for high-performance
+       | MinIO recommends a minimum of 8 drives per server
+
+   * - :octicon:`circle`
+     - | 25GbE network for capacity 
+       | 100GbE Network interface cards for high performance
+
+.. important:: 
+
+   Object storage operations are primarily limited by network throughput.
+   Additional performance limits can be imposed by the storage controller and the storage devices themselves. 
+
+   For example:
+
+   - 1GbE network link can support approximately 125MB/s of throughput - which may not support the full capabilities of even a single spinning disk
+   - 10GbE network can support approximately 1.25GB/s, potentially supporting 4-6 spinning disk drives running at ~200MB/s (such as Western Digital Gold drives)
+   - 25GbE network can support approximately 3.125GB/s, potentially supporting 12 disks
+
+   The recommended minimum MinIO cluster of 4 nodes with 4 drives each (16 total disks) requires a 50GbE network to support the total potential aggregate throughput.
+
+   MinIO takes full advantage of the modern hardware improvements such as AVX-512 SIMD acceleration, 100GbE networking, and NVMe SSDs when available.
+
+Recommended Hardware Tests
+--------------------------
+
+MinIO Diagnostics
+~~~~~~~~~~~~~~~~~
+
+Run the following command.
+If you have access to :ref:`SUBNET <minio-docs-subnet>`, you can upload the results there.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support diag ALIAS
+
+Drive Tests
+~~~~~~~~~~~
+
+Test each drive independently on all servers to ensure they are identical in performance.
+Use the results of these OS-level tools to verify the capabilities of your storage hardware.
+Record the results for later reference.
+
+#. Test the drive's performance during write opeartions
```

Comment:
```suggestion
#. Test the drive's performance during write operations
```

Reply:
You know, science has shown that the order of letters in a word is less important than the beginning and ending letters.

Fixed anyway, though. Just because.